frontend/src/components/user/SavedSearchQueryUserControl.tsx

```diff
@@ -0,0 +1,32 @@
+"use client";
```

Comment:
This file might be overkill for copy/paste, but would imagine using this for managing the user state and data fetching in the upcoming work.

Reply:
I don't really care TBH, updated to `SaveSearchPanel`

Reply:
did you push? I'm not seeing the change